src/main/resources/openshift/app/configuration/eap.cli

```diff
@@ -16,8 +16,7 @@ jms-topic add --topic-address=executorCancellation --entries=topics/executorCanc
 
 /subsystem=undertow/server=default-server/http-listener=default:write-attribute(name=max-post-size, value=943718400)
 
-/subsystem=undertow/configuration=handler/file=windup-web-redirect:add
-/subsystem=undertow/configuration=handler/file=windup-web-redirect:write-attribute(name=path,value=${jboss.home.dir}/windup-web-redirect)
+/subsystem=undertow/configuration=handler/file=windup-web-redirect:add(path=${jboss.home.dir}/windup-web-redirect)
 /subsystem=undertow/server=default-server/host=default-host/location=\//:write-attribute(name=handler,value=windup-web-redirect)
```

Comment:
Not sure if that is just me, but on every OCP installation I tested, I had to add mdb delivery groups.

## Messaging
if (outcome != succes) of /subsystem=ejb3/mdb-delivery-group=dg_executors:read-resource
  /subsystem=ejb3/mdb-delivery-group=dg_executors:add(active=true)
end-if
if (outcome != succes) of /subsystem=ejb3/mdb-delivery-group=dg_services:read-resource
  /subsystem=ejb3/mdb-delivery-group=dg_services:add(active=true)
end-if

Reply:
That should only be the case in master, where I expect to remove these soon. The new deployment is using the windup-openshift project.

Perhaps we can set up a meeting to go over how to use it? It deploys much more quickly than this as well.

Reply:
@jsight thanks! ff the readme is complete-ish, I can try from there and come back to your offer if I fail

Reply:
@jeichler - I think Marco did an excellent job in putting together a usable readme, so it should be complete enough to work from. I think that Phil was able to deploy from the readme, but please do let us know if you run into issues with it. I believe that @huffmanca will be building the docs using it as a guide, so any feedback will likely make that easier for him as well.